### DISCUSSION TITLE
running server using docker causing the server to load multiple times

### DISCUSSION THREAD

**Discussion by Mogady** [fastapi/fastapi#7146] [Questions] [answered]
### First check

* [x] I added a very descriptive title to this issue.
* [x] I used the GitHub search to find a similar issue and didn't find it.
* [x] I searched the FastAPI documentation, with the integrated search.
* [x] I already searched in Google "How to X in FastAPI" and didn't find any information.
* [x] I already read and followed all the tutorial in the docs and didn't find an answer.
* [x] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
* [x] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
* [x] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).
* [x] After submitting this, I commit to one of:
    * Read open issues with questions until I find 2 issues where I can help someone and add a comment to help there.
    * I already hit the "watch" button in this repository to receive notifications and I commit to help at least 2 people that ask questions in the future.
    * Implement a Pull Request for a confirmed bug.

### Example

```Python
from fastapi import FastAPI


app = FastAPI()


@app.get("/")
async def root():
    return {"message": "test"}
```
docker
```
FROM tiangolo/uvicorn-gunicorn-fastapi:python3.7
COPY requirements.txt /tmp/
RUN pip install -r /tmp/requirements.txt
COPY . /app
```

### Description

* build and run the docker .
```bash
docker build -f docker -t image .
docker run -t -i -p 80:80 image
```
* i get multiple Application startup complete logs and the RAM usage is increased by each one.
<img width="1620" alt="sc" src="https://user-images.githubusercontent.com/20929301/95643432-bd19c980-0a74-11eb-8319-4c291864dacb.png">

* expect to get only on instance of server running same as doing it without docker 
<img width="586" alt="Screen Shot 2020-10-09 at 9 17 58 PM" src="https://user-images.githubusercontent.com/20929301/95643460-edf9fe80-0a74-11eb-8b9d-b5326feff4bd.png">

### Environment

* OS: [macOS]:
* FastAPI Version [0.61.1]:

* Python version:
3.8

**Comment by Mause** [comment, marked as answer]
This is expected, gunicorn starts multiple uvicorn workers to serve requests

**Comment by Mogady** [comment]
Thanks got it

**Comment by tiangolo** [comment]
Thanks for the help @Mause! 🍰 

And thanks for coming back to close the issue @Mogady 🍪 

> Sorry for the long delay! 🙈  I wanted to personally address each issue/PR and they piled up through time, but now I'm checking each one in order.
